packages/core/src/lib/history.ts

```diff
@@ -0,0 +1,110 @@
+import { LogResult, simpleGit } from 'simple-git';
+import {
+  CoreConfig,
+  Format,
+  PersistConfig,
+  UploadConfig,
+  uploadConfigSchema,
+} from '@code-pushup/models';
+import { getCurrentBranchOrTag, safeCheckout } from '@code-pushup/utils';
+import { collectAndPersistReports } from './collect-and-persist';
+import { GlobalOptions } from './types';
+import { upload as uploadCommandLogic } from './upload';
+
+export type HistoryOnlyOptions = {
+  targetBranch?: string;
+  uploadReports?: boolean;
+  forceCleanStatus?: true;
```

Comment:
```suggestion
  forceCleanStatus?: boolean;
```

Reply:
Even if the typing was on porpose I understand the handling of booleans now different and would also always stick to the above.